src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,34 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
As described in common mistakes, you should divide [your logic](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#divide-program-logic) into separate methods. I'd recommend creating separate methods for reading, report generating, and writing